sources/ConflictWindow.cpp

```diff
@@ -19,7 +19,7 @@ static const unsigned kReplace = 'RPLC';
 static const unsigned kSkip = 'SKIP';
 
 ConflictWindow::ConflictWindow(const char* file)
-	: BWindow(BRect(0, 0, 0, 0), "", B_BORDERED_WINDOW, B_WILL_ACCEPT_FIRST_CLICK | B_AUTO_UPDATE_SIZE_LIMITS)
+	: BWindow(BRect(0, 0, 0, 0), "", B_MODAL_WINDOW, B_WILL_ACCEPT_FIRST_CLICK | B_AUTO_UPDATE_SIZE_LIMITS)
```

Comment:
While it isn't resizable, without B_NOT_RESIZABLE there's still the resizing corner. No need to show that, if the window doesn't resize anyway.

> I have also removed the code that sets the Skip button as the default button because the new window can never receive keyboard focus.

Yes. That's the slight drawback: it's not keyboard-navigatable anymore. While annoying in some situations, it's better having to grab for your mouse than unbeknownst trigger an action you didn't notice...